docusaurus.config.js

```diff
@@ -42,22 +42,14 @@ module.exports = {
               label: "Style Guide",
               to: "docs/",
             },
-            {
-              label: "Second Doc",
-              to: "docs/doc2/",
-            },
           ],
         },
         {
           title: "Community",
           items: [
             {
-              label: "Stack Overflow",
-              href: "https://stackoverflow.com/questions/tagged/docusaurus",
-            },
-            {
-              label: "Discord",
-              href: "https://discordapp.com/invite/docusaurus",
+              label: 'Discord',
```

Comment:
Remove this and put our slack channel link, please

Reply:
Is there a link for the main code bases Githubs?